src/checkers/inference/InferenceLauncher.java

```diff
@@ -388,6 +409,36 @@ private static String getJaifFilePath(File outputDir) {
         return filePaths;
     }
 
+    /**
+     * Determine the version of the JRE that we are currently running and select a jdkX where X is
+     * the version of Java that is being run (e.g. 8, 9, ...)
+     *
+     * @return "jdk<em>X</em>" where X is the version of Java that is being run (e.g. 8, 9, ...)
+     */
+    public static String getJdkJarPrefix() {
```

Comment:
These methods are still in `SystemUtil` https://github.com/typetools/checker-framework/pull/3264/files#diff-aace53775ed3a60921a295e85526326eR191
Why do they need to be duplicated here?